### PR TITLE
Fixes #421 - Changed python virtualenv to /opt/mozdef/envs/python in all cron/*.sh files

### LINKED ISSUE #421: Change python virtualenv to /opt/mozdef/envs/python
This means we need to modify each cron/*.sh file to point to the new python binary.

### PR DESCRIPTION
**Pull request by scriptonist**
Changed python virtualenv to /opt/mozdef/envs/python in all cron/*.sh files

### CODE REVIEW

File: cron/backupSnapshot.sh [hunk 1]
```diff
@@ -8,5 +8,5 @@
 # Contributors:
 # Anthony Verez averez@mozilla.com
 
-source  /opt/mozdef/envs/mozdef/bin/activate
-/opt/mozdef/envs/mozdef/cron/backupSnapshot.py -c /opt/mozdef/envs/mozdef/cron/backup.conf
+source  //opt/mozdef/envs/python/pythonpython/mozdef/bin/activate
```
Comment (pwnbus) [inline cron/backupSnapshot.sh]: Looks like a find replace error here

File: cron/vidyo2MozDef.sh [hunk 2]
```diff
@@ -8,6 +8,6 @@
 # Contributors:
 # Jeff Bryner jbryner@mozilla.com
 
-source  /opt/mozdef/envs/mozdef/bin/activate
-/opt/mozdef/envs/mozdef/cron/vidyo2MozDef.py -c /opt/mozdef/envs/mozdef/cron/vidyo2MozDef.conf
+source  //opt/mozdef/envs/python/pythonpython/mozdef/bin/activate
+//opt/mozdef/envs/python/pythonpython/mozdef/cron/vidyo2MozDef.py -c //opt/mozdef/envs/python/pythonpython/mozdef/cron/vidyo2MozDef.conf
```
[the remark this answers is not in the document]
Comment (Phrozyn) [reply cron/vidyo2MozDef.sh]: also there's a leading slash that shouldn't be there on the second line.

**Comment by scriptonist** [comment]
My bad ! Pushed the wrong branch !. Let me correct that ASAP

**Comment by pwnbus** [comment]
@scriptonist Since we haven't heard anything from ya for a month, we're gonna close this PR. 

Feel free to address the comments and commit the fixes, and we can either reopen this PR, or create a new one :)

**Pull request closed without merging**